packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -149,6 +149,21 @@ export namespace CommonCommands {
         category: VIEW_CATEGORY,
         label: 'Close All Tabs'
     };
+    export const CLOSE_EDITOR: Command = {
```

Comment:
we should rename constants as well, that there is no notion of editor, CLOSE_TAB_IN_MAIN or something like that